aiorpc/client.py

```diff
@@ -35,11 +36,12 @@ class RPCClient:
         Unpacker.
     """
 
-    def __init__(self, host, port, *, timeout=3, loop=None,
+    def __init__(self, host=None, port=None, path=None, timeout=3, loop=None,
                  pack_encoding='utf-8', unpack_encoding='utf-8',
                  pack_params=None, unpack_params=None):
         self._host = host
         self._port = port
+        self._path = path
```

Comment:
The one minor thing that I would add is an exception if both Host and Path are defined telling the user that they cannot have both

Reply:
I don't think that's necessary -- specifying both wouldn't break execution.